server/helpers/common_registry_logs.go

```diff
@@ -65,6 +65,8 @@ func HandleError(h v1beta1.Host, en entity.Entity, err error, isModelError bool,
 	switch entity := en.(type) {
 	case *v1beta1.ComponentDefinition:
 		entityName := "[ " + entity.Model.Name + " " + entity.Model.Model.Version + " ]" + "( " + entity.DisplayName + " )"
+		//TODO:  Move this outside this function - this function is supposed to handle errrors, not check
```

Comment:
I wonder if during the time that this is under review whether we can tie-off this to-do item before merge.

Reply:
That's fixed @leecalcote . We are not using that function anymore. I just forgot to remove that. I'll remove it.

Reply:
Oh, very good.

Reply:
Removed the comments